domains/nova.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "nova.is-a.dev",
+    "owner": {
+        "username": "LunarN0v4",
+        "email": "zeus@zeusteam.dev",
+        "discord": "7un4r"
+    },
+    "record": {
+        "CNAME": ["webmail.zeusteam.dev"]
```

Comment:
CNAME should just be a string, no array needed.

Reply:
Fixed in latest commit